lib/LINE/Bot/API/Client/Furl.pm

```diff
@@ -159,6 +159,31 @@ sub post_image {
     $ret;
 }
 
+sub post_file {
+    my ($self, $url, $body, $file_path) = @_;
+
+    open my $fh, '<', $file_path
+        or croak 'Failed to open file.';
+
+    my($res_minor_version, $res_status, $res_msg, $res_headers, $res_content) = $self->{furl}->post(
+        $url,
+        [
+            $self->credentials,
+            'Content-Type' => 'multipart/form-data',
+        ],
+        [
+            @$body,
+            'file' => $fh,
```

Comment:
I wonder if Furl would corretly anotate this as `text/plain` type with only a filehandle -- this needs to be doublechecked.

Reply:
https://developers.line.biz/en/reference/messaging-api/#create-upload-audience-group-by-file

```bash
curl -v -X POST https://api-data.line.me/v2/bot/audienceGroup/upload/byFile \
-H 'Authorization: Bearer {channel access token}' \
-F 'description=audienceGroupName' \
-F 'file=@audiences.txt;type=text/plain'
```

I'm referring to the `type=text/plain'` part from that example.

Reply:
@gugod 
Sorry for my replying late...
I think using multiple `-F` option on `curl` and it means `multipart/form-data`.
So I consider we can ignore `text/plain` for using `multipart/form-data`.
what do you think about this?